integration-tests/test-suite/partials/status-codes.xml

```diff
@@ -32,11 +32,24 @@
 		</compare>
 		<urls>
 			<url href="comparators/statuscodes/failed.jsp"/>
-			<url href="comparators/statuscodes/success.jsp"/>
 			<url href="comparators/statuscodes/noneexistingPage.jsp"/>
 		</urls>
 	</test>
 
+	<test name="F-comparator-StatusCodes-200-success" useProxy="rest">
```

Comment:
We have new test added here.
Please update also our integration tests (functional).